portfolio/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -0,0 +1,80 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Iterator;
+
+public final class FindMeetingQuery {
```

Comment:
Since you have a method named query, it's probably a better idea to name this something like "MeetingFinder" or something, since having Query both in the class and method name is a little redundant. You could also keep the class name and rename the method to "execute" or something, if the intent is for each instance of this class to only ever represent a single query.